srcpkgs/Solaar/template

```diff
@@ -11,7 +11,7 @@ maintainer="Young Jin Park <youngjinpark20@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://pwr-solaar.github.io/Solaar/"
 changelog="https://raw.githubusercontent.com/pwr-Solaar/Solaar/master/ChangeLog.md"
-distfiles="https://github.com/pwr-Solaar/Solaar/archive/${version}.tar.gz"
-checksum=f8ac1db77d2713f057a83e448d84f6266d63c4fd36235ddb3dc20c6c8cbc0efe
+distfiles="${PYPI_SITE}/s/solaar/solaar-${version}.tar.gz"
+checksum=04224eec598a7b5c988d52e823a2257cb0255257736b915202388c5aafc0fd9c
 # Package provides no tests
 make_check=no
```

Comment:
If the package provides no tests, there shouldn't be anything that can fail, so we don't have to skip them, no?

Reply:
The setup.py testing procedure always errs out when there are no tests defined. I don't quite understand why it happens but it's the same for other `python3-module` packages (with no tests) too as far as I have seen.

Reply:
It only fails because it checks for dependencies:
```
=> Solaar-1.1.7_1: running do_check ...
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
/usr/lib/python3.11/site-packages/setuptools/installer.py:27: SetuptoolsDeprecationWarning: setuptools.installer is deprecated. Requirements should be satisfied by a PEP 517 installer.
  warnings.warn(
WARNING: The wheel package is not available.
/usr/bin/python3: No module named pip
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpkph0cl5h', '--quiet', 'typing_extensions>=4.0.0']' returned non-zero exit status 1.
=> ERROR: Solaar-1.1.7_1: do_check: '${make_check_pre} python3 setup.py ${make_check_target} ${make_check_args}' exited with 1
=> ERROR:   in do_check() at common/build-style/python3-module.sh:28
```
So removing `make_check=no`, adding `checkdepends="$depends"` and patching out the typing_extensions dependency (import this patch https://github.com/pwr-Solaar/Solaar/commit/e15b58fdaa9137556e98e618489dffac12de13b8.patch) will make it pass. This is not the same as not running tests because it checks dependencies which is useful. Adding typing_extensions to checkdepends would also fix it, but we will probably forget to remove it next time the package is updated and a patch will stop applying, so that's better.